app/controllers/teams_controller.rb

```diff
@@ -2,13 +2,8 @@
 
 class TeamsController < ApplicationController
   before_action :authenticate_user!
-  before_action -> { redirect_to_root_unless_user(:can_manage_teams?) }, except: [:edit, :update]
```

Comment:
I think this safeguard should stay in place, with an adequate replacement of `can_manage_teams?`

Reply:
This safeguard was for `def index`, but the `def index` won't exist after this PR is merged, that is the reason I removed it. There is already safeguard for other actions in the next line.

Reply:
Ah okay, in that case I misread something in the controller. Thanks for confirming!